app/assets/javascripts/discourse/app/modifiers/trap-tab.js

```diff
@@ -13,8 +13,9 @@ export default class TrapTabModifier extends Modifier {
     registerDestructor(this, (instance) => instance.cleanup());
   }
 
-  modify(element, [options]) {
-    this.preventScroll = options?.preventScroll ?? true;
+  modify(element, _, { preventScroll, autofocus }) {
+    autofocus ??= true;
```

Comment:
I would want to write a test for this, but I realised we have never been written much tests for modifiers. I will try to follow-up but I need to fix regression before.